### DISCUSSION TITLE
Aiuto nel registro

### DISCUSSION THREAD

**Discussion by Giuppimack65** [iisgiua/giuaschool#679] [Richieste di aiuto]
salve, ho fatto come mi ha indicato ma alla fine c'è sempre lo stesso errore, come faccio, magari c'è lo può fare lei il registro??? attendo un vostro riscontro

**Comment by CobraOfficial01** [comment]
@trinko

**Comment by trinko** [comment]
> salve, ho fatto come mi ha indicato ma alla fine c'è sempre lo stesso errore, come faccio, magari c'è lo può fare lei il registro??? attendo un vostro riscontro

No, io non svolgo questo servizio.
Se descrivi con tutti i dettagli il problema, posso aiutarti a risolvere.

**Comment by CobraOfficial01** [comment]
![image](https://github.com/iisgiua/giuaschool/assets/128899928/66bff81c-11bd-4871-a463-0fb84e2b5ce8)

**Comment by CobraOfficial01** [comment]
questo è il problema @trinko

**Comment by trinko** [comment]
Controlla la versione PHP che stai usando sul tuo hosting e impostala sulla versione 8.0

**Comment by CobraOfficial01** [comment]
ho messo su php 8.0

**Comment by trinko** [comment]
Se l'errore è sempre lo stesso indicato, devo suupporre che non è stato effettuato correttamente il cambio di versione PHP sul server.

L'errore riportato è molto specifico (errore di sintassi) e può dipendere solo dall'uso del PHP con versione precedente alla 7.4.

Se stai usando Altervista, la procedura per il cambio di versione è questa:

- Vai al pannello di controllo di Altervista, quindi sul menu RISORSE.
- Cerca il riquadro PHP e clicca sul collegamento "Modifica Impostazioni PHP".
- A questo punto cerca l'impostazione della versione PHP e impostala su "PHP8" e clicca sul pulsante SALVA.

**Comment by CobraOfficial01** [comment]
ho risolto grazie
